test/functional/scan/test-scan-exclusive-tbb.cpp

```diff
@@ -0,0 +1,28 @@
+
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2016-20, Lawrence Livermore National Security, LLC
+// and RAJA project contributors. See the RAJA/COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#include "test-scan-exclusive.hpp"
+
+#if defined(RAJA_ENABLE_TBB)
+
+// TBB policy types to test
+using TBBExecTypes = list< RAJA::tbb_for_exec,
+                           RAJA::tbb_for_static< >,
```

Comment:
We might not need this one since tbb_for_exec aliases tbb_for_static<>

Reply:
I thought about that. Then, I thought maybe in the future that may not be the case so I added it. It's not horribly redundant.